src/database/services/keycloak-api-service.ts

```diff
@@ -75,7 +78,110 @@ export const CreateKeycloakApiService = (): KeycloakApiService => {
         severaUserId: (user as CustomKeycloakProfile).severaUserId ?? undefined,
       } as CustomKeycloakProfile;
     },
-  };
+
+    /**
+     * Updates a user's attributes
+     * 
+     * @param id string
+     * @param attributes  Record<string, string[]>
+     */
+    updateUserAttribute: async (
+      id: string,
+      attributes: Record<string, string[]> 
+    ): Promise<void> => {
+      const bodyContent = {
+        attributes: attributes,
+      };
+
+      try {
+        const response = await fetch(
```

Comment:
I suppose as we are not returning from these services maybe typing is not essential. However please do some additional investigation about types from keycloak as it would be very nice to have a valid type for the fetch response from a given endpoint.

Reply:
If you do find a way to type the fetch response, please update all fetch response types in the service